include/volasim/args.h

```
@@ -17,6 +35,32 @@ inline Args parseArgs(int argc, char* argv[]) {
         throw std::runtime_error(arg + " requires a path argument");
       }
       args.world_path = argv[++i];
+    } else if (arg == "--physics-hz") {
+      if (i + 1 >= argc) {
+        throw std::runtime_error(arg + " requires a rate in Hz");
+      }
+
+      const std::string value = argv[++i];
```

Clang (14.0.6)

[error] variable 'std' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)